lib/tasks/workers.rake

```diff
@@ -10,4 +10,15 @@ namespace :workers do
       Sneakers::Runner.new(workers).run
     end
   end
+
+  namespace :delete_folders do
+    desc 'run a FolderDeletionJob'
+    task run: :environment do
+      silence_warnings do
+        Rails.application.eager_load! unless Rails.application.config.eager_load
+      end
+      workers = [ FolderDeletionJob.job_wrapper ]
```

Comment:
I'm surprised that this isn't causing tests to fail, since FolderDeletionJob no longer exists...